src/aequitas/flow/methods/postprocessing/group_threshold_new.py

```diff
@@ -0,0 +1,178 @@
+from typing import Optional, Union
+
+import numpy as np
+import pandas as pd
+
+from ...utils import create_logger
+from .postprocessing import PostProcessing
+from .threshold import Threshold
+
+
+class GroupThreshold(PostProcessing):
+    def __init__(
+        self,
+        threshold_type: str,
+        threshold_value: Union[float, int],
+        fairness_metric: str,
+    ):
+        """Initialize a new instance of the GroupThreshold class.
+
+        Parameters
+        ----------
+        threshold_type : str
+            The type of threshold to apply. It can be one of the following:
+                - fpr: applies a threshold to obtain a specific false positive rate.
+                - tpr: applies a threshold to obtain a specific true positive rate.
+                - top_pct: applies a threshold to obtain the top percentage of
+                           predicted scores.
+                - top_k: applies a threshold to obtain the top k predicted scores.
+        threshold_value : Union[float, int]
+            The value to use for the threshold, depending on the threshold_type
+            parameter.
+        fairness_metric : str
+            The metric to use for measurement of fairness. It can be one of the
+            following:
+                - tpr: true positive rate
+                - fpr: false positive rate
+                - pprev: predicted prevalence
+        """
+        self.logger = create_logger("methods.postprocessing.BalancedGroupThreshold")
```

Comment:
Isn't it supposed to be just GroupThreshold?

Reply:
Yes it is! I have not settled for a name still xD